README.md

```diff
@@ -14,6 +14,20 @@ Although this project is very new, it has been thoroughly tested in all modern b
 
 __[Demo](http://jakesidsmith.github.io/react-reorderable/)__
 
+## Installation
+
+* Using npm
+  ```
+  npm install react-reorder
+  ```
+    Add `--save` or `-S` to update your package.json
+
+* Using bower
+  ```
+  bower install react-reorderable
```

Comment:
Hm, that wouldn't be good. I just wonder if having two names won't be confusing to people. Maybe there's a way to deprecate the old name? I've been considering a name change for one of my libraries, too, and I'm not sure what the best way is to go about it.